README.md

```diff
@@ -1,51 +1,50 @@
+West Oakland Air Quality (WOAQ) is a project of [OpenOakland](https://www.openoakland.org/) focused on building digital advocacy tools around air quality data collected by volunteers and citizen scientists.
 
-Project details
----------------
-See the [Project Overview](http://bit.ly/WOAQoverview) doc for details about our partner, project history, and more info about air quality issues in West Oakland. 
+WOAQ partners with [West Oakland Environmental Indicators Project](https://www.woeip.org) (WOEIP), a community-based environmental justice organization that has been collecting air quality data for over a decade. WOEIP is dedicated to achieving healthy homes, healthy jobs and healthy neighborhoods for all who live, work, learn and play in West Oakland, California.
 
-Technology used
----------------
-- [Django](https://www.djangoproject.com/) (Python 3.0 framework)
-- [MySQL](https://www.mysql.com/) (database)
-- [Zurb Foundation 6](https://foundation.zurb.com/sites.html) (front-end framework)
-- [Docker](https://www.docker.com/) (optional, for running locally)
+To learn more about the history of WOEIP, the roadmap for this project and more, check out the [About](https://github.com/openoakland/woeip/blob/master/.github/about.md) section.
 
-How To contribute
------------------
-We welcome new contributors and there are several ways you can do so. If you decide to join us, we ask that you agree to adhere to the OpenOakland [Code of Conduct](https://openoakland.org/code-of-conduct/) and the [17 Principles of Environmental Justice](https://www.ejnet.org/ej/principles.html).
+# Quickstart
 
-### Development options
-- Add a comment to any [open issue](https://github.com/openoakland/woeip/issues?utf8=%E2%9C%93&q=is%3Aopen++). Or;
-- Clone the repo, run the site locally, create a branch, and get to work on any open issue labelled MVP and Ready. See [contributing.md](https://github.com/openoakland/woeip/tree/master/.github/contributing.md) for full details. 
+Run the woaq infrastructure with [Docker Compose](https://docs.docker.com/compose/)
 
-### Design & research options
-We use [Trello](https://trello.com/b/EBnxZHmx/west-oakland-air-quality) for project management:
-- Review and add a comment to any light blue “Design & Research” card currently sitting in the [In QA](https://trello.com/b/EBnxZHmx/west-oakland-air-quality) column. Or;
-- Grab any card tagged with the light blue “Design & Research” label in the [Ready to Work On](https://trello.com/b/EBnxZHmx/west-oakland-air-quality) column and get started. 
+```bash
+  git clone https://github.com/openoakland/woeip.git
+  cd woeip
+  docker-compose up -d --build
```

Comment:
Since the db name has changed, I had to create a new database `woaq`, so I would suggest adding that in the README as an instruction for initial setup. This was the way I did it:
```
docker exec -it woaq-db /bin/bash
psql -U postgres
CREATE DATABASE woaq;
```

Reply:
Talking to @ifed3 now about this. When I set this up fresh, I didn't have to create the database manually, but worth looking into what was different in your case. One solution would be to have an entrypoint script that does this.

Reply:
Opened #126 to address.